roles/rac-db-setup/tasks/rac-db-install.yml

```diff
@@ -153,10 +153,8 @@
 - name: rac-db-install | opatch output
   debug:
     msg:
-      - "{{ apply_oneoff.cmd }}"
```

Comment:
I ran into the issue here too.  But we want to avoid printing out pages of patches that were skipped.  So instead, I'd suggest keeping the existing looping, but referencing - "{{ item.cmd }}" and "{{ item.stdout_lines }}"